### ISSUE TITLE
Finish the prep-to-measurement animation for the multi-coin quantum case

### ISSUE THREAD

**Issue by jbphet** [phetsims/quantum-measurement#16]
In order to have something that looked reasonable for the NSF demo on 6/27/2024 I created a quick version of the animation for moving the quantum coins from the prep area to the measurement area.  As of this writing, that animation doesn't depict the coins in their transposed state as they travel - it just shows them as masked (gray).  It also doesn't do the thing that the single coin test box does where it has a mask that appears on the coin as it moves into the test box.  I'll need to add these elements after the demo version is released.

**Comment by jbphet** [comment]
@AgustinVallejo and I were discussing this today, and he said he thought that in a design meeting that occurred last week (while I was out), the designers said that it would be okay on the lower animation for the coins to be gray (aka masked) during the entire animation, and not become masked as they slide into the measurement area as happens with the single coin.  @arouinfar - are you okay with this?

**Comment by arouinfar** [comment]
> @AgustinVallejo and I were discussing this today, and he said he thought that in a design meeting that occurred last week (while I was out), the designers said that it would be okay on the lower animation for the coins to be gray (aka masked)

Yes, that's correct. We discussed in the 9/4/24 design meeting and decided to continue with the masked coins for the multiple coin animation for a few reasons.
* Due to the smaller size, we would need to display the coins with a solid color (as when revealed). This could look odd when animated directly from the full-size coin.
* For the Quantum case, even if we use the correct color of the superposition, it can suggest that we are "observing" the coins as they transition over to the experiment area, which is problematic.
* We could create a pipe or other device to transmit the coins through, so the animation distance is shorter, but this requires a more elaborate design, which is out-of-scope.

> Regardless, we will still want to improve the animation so that the coins spread out gradually as they travel to the measurement area.

Back to you @jbphet.

**Comment by AgustinVallejo** [comment]
We reviewed this at 2025-02-04 design meeting and decided this was done
